OpenRA.Mods.Common/Traits/Attack/AttackSuicides.cs

```diff
@@ -74,11 +74,12 @@ public void ResolveOrder(Actor self, Order order)
 				if (!order.Queued)
 					self.CancelActivity();
 
-				self.SetTargetLine(target, Color.Red);
-
 				self.QueueActivity(move.MoveToTarget(self, target));
+				self.ShowTargetLines();
 
-				self.QueueActivity(new CallFunc(() => self.Kill(self)));
+				var act = new CallFunc(() => self.Kill(self));
+				act.SetTargetLineNode(target, Color.Red, true);
```

Comment:
Likewise here: the `MoveToTarget` should already be drawing the target line, so why repeat it?

Reply:
MoveToTarget is supposed to be used in conjunction with many activities so sometimes, it is red (C4 bombing), sometimes it is yellow (engi capture) so this is not repeating.